src/moov/trak/mdia/minf/stbl/stsd/h264/avc3.rs

```diff
@@ -0,0 +1,76 @@
+use super::avc1::AvcSampleEntry;
+
+const AVC3_CODE: u32 = u32::from_be_bytes(*b"avc3");
+
+pub type Avc3 = AvcSampleEntry<{ AVC3_CODE }>;
+
+#[cfg(test)]
+mod tests {
```

Comment:
These tests are OK, but a check of decoding a real `avc3` box (from whatever source motivated the need for this change) would be useful. 

If that isn't achievable, or you'd like to have more than one, you could pull that box out of one of the MPEG file format conformance samples, as shown here https://gpac.github.io/mp4box.js/test/filereader.html?https://mpeggroup.github.io/FileFormatConformance/files/published/nalu/l-hevc/lhevc_avc3_lhv1.mp4